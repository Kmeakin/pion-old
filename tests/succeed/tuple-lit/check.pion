//~ run = "${PION} elab ${FILE}"

{
let _ = ();
let _ = (1,);
let _ = (1, 2);
let _ = (1, 2, 3);

let _ : Type = ();
let _ : Type = (Int,);
let _ : Type = (Int, Int);
let _ : Type = (Int, Int, Int);

{}
}
