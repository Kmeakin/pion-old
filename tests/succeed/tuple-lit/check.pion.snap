exit status: 0
stdout:
let _ : () = ();
let _ : (Int,) = (1,);
let _ : (Int, Int) = (1, 2);
let _ : (Int, Int, Int) = (1, 2, 3);
let _ : Type = ();
let _ : Type = (Int,);
let _ : Type = (Int, Int);
let _ : Type = (Int, Int, Int);
() : ()

stderr:
